src/main/java/mekanism/common/integration/OreDictManager.java

```diff
@@ -352,18 +345,31 @@ public static void init(RegistryEvent.Register<IRecipe> event) {
             ItemStack dustCoal = StackUtils.size(oreDict.get(0), 1);
             RecipeHandler.addCrusherRecipe(new ItemStack(Items.COAL), dustCoal);
         }
+        for (ItemStack dust : oreDict) {
```

Comment:
Move inside if statement, see one of my other comments.

Reply:
ok did that.